lightning/src/util/macro_logger.rs

```diff
@@ -41,24 +42,26 @@ macro_rules! log_bytes {
 pub(crate) struct DebugFundingChannelId<'a>(pub &'a Txid, pub u16);
 impl<'a> core::fmt::Display for DebugFundingChannelId<'a> {
 	fn fmt(&self, f: &mut core::fmt::Formatter) -> Result<(), core::fmt::Error> {
-		(OutPoint { txid: self.0.clone(), index: self.1 }).to_channel_id().fmt(f)
+		ChannelId::v1_from_funding_outpoint(OutPoint { txid: self.0.clone(), index: self.1 }).fmt(f)
```

Comment:
My gut feeling here is that channelID could be passed to `DebugFundingChannelId`, so no implicit decision to use v1 channelID. `DebugFundingChannelId` & `log_funding_channel_id` have very limited use, easy to change.